js/webviewMenu.js

```diff
@@ -194,6 +194,18 @@ const webviewMenu = {
     }
 
     if (link || (mediaURL && !mediaURL.startsWith('blob:'))) {
+      if (link.startsWith('mailto:')) {
+        clipboardActions.push({
+          label: l('copyEmailAddress'),
+          click: function () {
+            var ematch = link.match(/(?<=mailto:)[^\?]+/)
+            if (ematch) {
+              clipboard.writeText(ematch[0])
+            }
+          }
+        })
+      }
+
       clipboardActions.push({
```

Comment:
Maybe put this inside an `else` block, so that "copy link" doesn't show up for email addresses?